examples/storage.py

```diff
@@ -4,6 +4,13 @@
 import asab.storage
 
 
+'''
+../etc/site.conf
+
+[asab:storage]
+type=mongodb
+
+'''
 class MyApplication(asab.Application):
```

Comment:
The name of the example should be `mongodb_storage.py`, see `inmemory_storage.py`